### PR TITLE
Update overview.mdx

### PR DESCRIPTION
**Pull request by piotrekkaminski**
More updates based on the review

### CODE REVIEW

File: docs/api-docs/users/overview.mdx [hunk 1]
```diff
@@ -2,46 +2,44 @@
...
-A user is an account that can sign in to the BigCommerce control panel for an account or store.
+A user is an account (email address) that can sign in to the BigCommerce control panel for an account or store.
 
 ## Prerequisites
 
 Before getting started with the GraphQL Users API, make sure you have the following:
 
 * BigCommerce store
-* A valid API account access token with sufficient permissions to make the desired requests
+* A valid API account access token with sufficient permissions to make the desired requests (see Getting Started below)
```
Comment (bc-tgomez) [inline docs/api-docs/users/overview.mdx:16]:
```suggestion
* A valid API account access token with sufficient permissions to make the desired requests (more on that in the  **Getting Started** segment below)
```

File: docs/api-docs/users/overview.mdx [hunk 2]
```diff
@@ -92,7 +90,7 @@ The following query returns details about an account:
 
 ### Account users
 
-The following query returns details about an account's users:
+The following query returns details about an account's users. Note a standard cursor-based pagination is supported:
```
Comment (bc-tgomez) [inline docs/api-docs/users/overview.mdx:93]:
```suggestion
The following query returns details about an account's users. Note that a standard cursor-based pagination is supported.
```

**Review by bc-tgomez — CHANGES_REQUESTED** [review]
This is great! Could you address the suggestions? Thanks.

**Comment by piotrekkaminski** [comment]
@bc-tgomez accepted the suggestions. thanks!

**Pull request merged**